frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-delete.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { modal } from '../../../integration-tests-cypress/views/modal';
+import {
+  poolName,
+  scName,
+  createBlockPool,
+  navigateToBlockPool,
+  blockPoolFooter,
+} from '../views/block-pool';
+import { pvc } from '../views/pvc';
+import { createStorageClass, deleteStorageClassFromCli } from '../views/storage-class';
+import { POOL_PROGRESS } from '../../src/constants/storage-pool-const';
+
+const pvcName: string = 'testing-pvc';
+
+describe('Test block pool deletion under OCS UI', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Check for a pool deletion', () => {
+    createBlockPool();
+
+    cy.log('Create storage class using newly created pool');
+    createStorageClass(scName, poolName);
+
+    cy.log('Create PVC using newly created storage class');
+    cy.clickNavLink(['PersistentVolumeClaims']);
+    pvc.createPVC(pvcName, '1', scName);
+    cy.visit('/');
+
+    cy.log('Delete a newly created block pool');
+    navigateToBlockPool();
+    cy.byLegacyTestID('kebab-button')
+      .first()
+      .click();
+    cy.byTestActionID('Delete Block Pool').click();
+
+    modal.modalTitleShouldContain('Delete Block Pool');
+    blockPoolFooter(POOL_PROGRESS.BOUNDED);
+
+    cy.log('Delete pvc and try pool deletion');
+    cy.exec(`oc delete PersistentVolumeClaim ${pvcName} -n openshift-storage`);
+
+    cy.byLegacyTestID('kebab-button')
+      .first()
+      .click();
+    cy.byTestActionID('Delete Block Pool').click();
+    blockPoolFooter('delete');
+
+    cy.log('Trying to delete default pool');
```

Comment:
This can be a different test case - 'Deleting the default block pools should fail'

Reply:
ack